Networking/Networking/Model/ShippingLabel/Enums/ShippingLabelPaymentCardType.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+/// The supported card types for Shipping Label Payment Methods.
+///
+public enum ShippingLabelPaymentCardType: GeneratedFakeable {
```

Comment:
We can use simple decodable enum here instead of custom `RawRepresentable` conformance.
Any reasons against that I'm missing?

```suggestion
public enum ShippingLabelPaymentCardType: String, Decodable, GeneratedFakeable {
```

It will also require a change in `Decodable` extension for `ShippingLabelPaymentMethod`:
```swift
let cardType = try? container.decode(ShippingLabelPaymentCardType.self, forKey: .cardType)
```

Reply:
Oh yes, that's much simpler, thanks! I don't think there's any reason not to, at least at this point, so I made that change in f1b2d53.